database/migrations/community/2024_03_22_00000_update_set_requests_table.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class() extends Migration {
+    public function up(): void
+    {
+        Schema::table('SetRequest', function (Blueprint $table) {
+            $table->dropColumn('User');
+        });
```

Comment:
This fails for me. And the column clearly does exist.
```
2024_03_22_00000_update_set_requests_table ............................................. 2ms FAIL

   Illuminate\Database\QueryException 

  SQLSTATE[42000]: Syntax error or access violation: 1072 Key column 'User' doesn't exist in table (Connection: mysql, SQL: alter table `SetRequest` drop `User`)
```

```
MariaDB [retroachievements-web]> desc SetRequest;
+------------+---------------------+------+-----+---------------------+----------------+
| Field      | Type                | Null | Key | Default             | Extra          |
+------------+---------------------+------+-----+---------------------+----------------+
| id         | bigint(20) unsigned | NO   | PRI | NULL                | auto_increment |
| user_id    | bigint(20) unsigned | YES  | MUL | NULL                |                |
| User       | varchar(32)         | NO   | MUL | NULL                |                |
```

Reply:
Ah, there's a key that had to be dropped too.
```
UNIQUE KEY `user_game_list_entry_username_game_id_type_unique` (`User`,`GameID`,`type`),
```

```
alter table `SetRequest` drop key `user_game_list_entry_username_game_id_type_unique`;
```

Reply:
Whoops, I missed that one. I've updated the migration to drop that unique key. This issue should be resolved now.